doc/source/pandas_on_ray.rst

```diff
@@ -0,0 +1,71 @@
+Pandas on Ray
+======
+
+Pandas on Ray is an early stage DataFrame library that wraps Pandas and
+transparently distributes the data and computation. The user does not need to
+know how many cores their system has, nor do they need to specify how to
+distribute the data. In fact, users can continue using their previous Pandas
+notebooks while experiencing a considerable speedup from Pandas on Ray, even
+on a single machine. Only a modification of the import statement is needed, as
+we demonstrate below. Once you’ve changed your import statement, you’re ready
+to use Pandas on Ray just like you would Pandas.
+
+.. code-block:: python
+
+  #import pandas as pd
+  import ray.dataframe as pd
+
+Currently, we have part of the Pandas API implemented and are working toward
+full functional parity with Pandas.
+
+Using Pandas on Ray on a Single Node
+------------------------------
+
+In order to use the most up-to-date version of Pandas on Ray, please follow
+the instructions on the `installation page`_
+
+Once you import the library, you should see something similar to the following
+output:
+
+.. code-block::
```

Comment:
I'd remove this code block, but up to you.

Reply:
I have it in there because if they have installed with `pip install ray` it won't show this. I only add this because many people keep opening issues on it.